package.json

```diff
@@ -5,6 +5,7 @@
     "yarn": ">=1.7.0"
   },
   "dependencies": {
+    "@github/webauthn-json": "^0.1.2",
```

Comment:
`v0.2.0` just released, https://github.com/github/webauthn-json/releases/tag/v0.2.0.

Changes `base64` to `base64url` :-)